api/src/main/scala/Utils.scala

```diff
@@ -72,6 +73,26 @@ trait Utils {
   }
 
   /**
+    * Given an optional boolean value, if it is true, returns Rasterizer
+    * Options treating the raster pixels as areas. If false, returns Options
+    * treating the raster pixels as points. If not specified, return default
+    * value, which treats pixels as points.
+    *
+    * @param   pixelIsArea  An optional boolean value
+    * @return               Rasterizer Options
+    */
+  def getRasterizerOptions(pixelIsArea: Option[Boolean]): Options =
```

Comment:
This is not something under our control here but I think `Options` would benefit from being renamed as `RasterizerOptions` or something more specific to more markedly distinguish it from `Option`.

Reply:
Pushed a fixup 197e107 that should take care of this.